src/js/utils/date-utils.js

```diff
@@ -1,48 +1,39 @@
+/*
+eslint no-underscore-dangle: ["error", {
```

Comment:
I spoke to @bjacobel about this and he reminded me that we explicitly lint against using leading and trailing underscores! Sorry for the conflicting advice, but let's remove them again. 

In fact, they aren't necessary at all in this class, because we are explicitly choosing what to make public in the returned object. If there are things we don't want folks to see, we just shouldn't return them.